crates/astria-sequencer/src/main.rs

```diff
@@ -1,17 +1,26 @@
 use astria_sequencer::{
-    Config,
+    config,
     Sequencer,
 };
 use tracing::info;
 
+// Following the BSD convention for failing to read config
+// See here: https://freedesktop.org/software/systemd/man/systemd.exec.html#Process%20Exit%20Codes
+const EX_CONFIG: i32 = 78;
+
 #[tokio::main]
 async fn main() {
-    let config = Config::get();
-    telemetry::init(std::io::stdout, config.log.as_deref().unwrap_or("info"))
-        .expect("failed to initialize telemetry");
+    let config = match config::get() {
+        Ok(cfg) => cfg,
+        Err(e) => {
+            eprintln!("failed to read configuration: {e:?}");
```

Comment:
```suggestion
            eprintln!("failed to read configuration:\n{e:?}");
```

Reply:
The newline would make this nicer to read..

Reply:
Oh sorry mb - oversight